### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by nitrag** [airbnb/lottie-ios#2181] [General]
Is it time for a new release? Your documentation is out of sync. I had to use Cocoapods against master for the fancy SwiftUI features. 

🍻

**Comment by calda** [comment]
Yes, a new release is coming very soon. Since this version adds support for visionOS, we have to wait until the Xcode 15 release candidate is published and available on GitHub actions. Once that is available I plan on releasing 4.3.0. Should be soon.
